Tests/XcodeprojTests/GenerateXcodeprojTests.swift

```diff
@@ -43,8 +43,34 @@ class GenerateXcodeprojTests: XCTestCase {
         }
 #endif
     }
-    
+
+    func testXcconfigOverrideValidatesPath() {
+#if os(macOS)
```

Comment:
Agreed; though there isn't much use for it, there's no reason that the project generation can't run on Linux.  Thanks for adjusting this.